idx/idx.go

```diff
@@ -29,6 +34,43 @@ type Archive struct {
 	LastSave uint32 // last time the metricDefinition was saved to a backend store (cassandra)
 }
 
+type MetricID struct {
+	org   int
```

Comment:
sidenote: i wish we could make this a uint also. see #260 #technicaldebt

Reply:
that seems reasonable, but not in the scope of this PR as it is big enough already